index.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable */
+import { Render } from "./modules/render.js";
+import { DateTime } from "./luxon.min.js";
+
+Render.renderBooks();
+
+const booksSection = document.querySelector('.books');
+const addSection = document.querySelector('.add-book');
+const contactSection = document.querySelector('.contact-section');
+const list = document.querySelector('.list');
+const addNew = document.querySelector('.add-new');
+const contact = document.querySelector('.contact');
+
+function showBooks() {
+  booksSection.style.display = 'flex';
+  addSection.style.display = 'none';
+  contactSection.style.display = 'none';
+};
```

Comment:
- [ ] Kindly use arrow syntax to declare all your functions in this file. Remember that part of the requirement in this milestone is to make you familiar with ES6 syntax. 👍